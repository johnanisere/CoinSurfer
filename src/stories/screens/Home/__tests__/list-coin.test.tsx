import * as React from "react";
import "react-native";
import ListCoin from "../list-coin";
// Note: test renderer must be required after react-native.
import * as renderer from "react-test-renderer";

const list = { map: jest.fn() };
const fetchCoin = jest.fn();

it("renders correctly", () => {
  const tree = renderer
    .create(<ListCoin list={list} fetching={false} fetchCoin={fetchCoin} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
