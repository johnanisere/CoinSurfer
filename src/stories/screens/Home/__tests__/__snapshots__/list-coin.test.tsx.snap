// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  contentContainerStyle={
    Array [
      Object {
        "padding": undefined,
      },
      undefined,
    ]
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={true}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardShouldPersistTaps="handled"
  keyboardSpace={0}
  onScroll={[Function]}
  refreshControl={
    <RefreshControlMock
      onRefresh={[MockFunction]}
      refreshing={false}
    />
  }
  resetKeyboardSpace={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
  viewIsInsideTabBar={false}
>
  <RCTRefreshControl />
  <View>
    <View
      closeOnRowBeginSwipe={false}
      closeOnRowPress={true}
      closeOnScroll={true}
      directionalDistanceChangeThreshold={2}
      disableLeftSwipe={false}
      disableRightSwipe={false}
      leftOpenValue={0}
      previewFirstRow={false}
      recalculateHiddenLayout={false}
      rightOpenValue={0}
      style={Object {}}
      swipeToOpenPercent={50}
    />
  </View>
</RCTScrollView>
`;
